### ISSUE TITLE
New map came out.

### ISSUE THREAD

**Issue by NewGitHubUser0** [w4rum/squadlanes#60]
Only if you have free time, please update the website to have RAAS layers of new map.

**Comment by D4n2021** [comment]
See https://github.com/w4rum/squadlanes/issues/58#issuecomment-1925769705 
Squadmaps is more feature rich and squadlanes is becoming deprecated, is the sad truth...

**Comment by NewGitHubUser0** [comment]
> See [#58 (comment)](https://github.com/w4rum/squadlanes/issues/58#issuecomment-1925769705) Squadmaps is more feature rich and squadlanes is becoming deprecated, is the sad truth...
Appreciate your answer.

**Comment by w4rum** [comment]
I'm still doing maintenance on the project for the time being and I'm planning on adding the map.

I don't have access to my desktop system until saturday, so this will take a bit still.

Anyway, I'll keep the issue open so I'm less likely to forget about it.

**Comment by grook8958** [comment]
> See [#58 (comment)](https://github.com/w4rum/squadlanes/issues/58#issuecomment-1925769705) Squadmaps is more feature rich and squadlanes is becoming deprecated, is the sad truth...

that's annoying, squadmaps is closed source

**Comment by D4n2021** [comment]
> > See [#58 (comment)](https://github.com/w4rum/squadlanes/issues/58#issuecomment-1925769705) Squadmaps is more feature rich and squadlanes is becoming deprecated, is the sad truth...
> 
> that's annoying, squadmaps is closed source

Doesn't seem so, it's on GitHub aswell after all https://github.com/mahtoid/SquadMaps

**Comment by grook8958** [comment]
> > > See [#58 (comment)](https://github.com/w4rum/squadlanes/issues/58#issuecomment-1925769705) Squadmaps is more feature rich and squadlanes is becoming deprecated, is the sad truth...
> > 
> > 
> > that's annoying, squadmaps is closed source
> 
> Doesn't seem so, it's on GitHub aswell after all https://github.com/mahtoid/SquadMaps

Nope, if you look at the repo it hasn’t been updated for months even though the latest map introduced in v7.1 is on squad maps. If you dig further you see the repo is completely outdated compared to the website

**Comment by D4n2021** [comment]
Oh, interesting. And disconcerting.

**Comment by mahtoid** [comment]
@grook8958 We plan to go open-source soon, just a lot of code to tidy up first in all honesty. It's pretty much monkey code right now. 

@D4n2021 `disconcerting` is a funny word to use.
